utils/routes/root/paramList.ts

```diff
@@ -1,8 +1,5 @@
-import { NativeStackScreenProps } from '@react-navigation/native-stack';
-import { ParamList as TabParamList } from '../tabs/paramList';
-
 export type ParamList = {
-  Tabs: NativeStackScreenProps<TabParamList>;
+  Tabs: undefined;
```

Comment:
Not sure why this was changed to undefined, but I wrote the wrong thing either way, it should be NavigatorScreenParams\<TabParamList\>